MH4U Dex/Images.xcassets/Region Icons/ancestral_steppe-icon.imageset/Contents.json

```diff
@@ -0,0 +1,21 @@
+{
+  "images" : [
+    {
+      "idiom" : "universal",
+      "scale" : "1x",
+      "filename" : "Image.png"
```

Comment:
Oh!   Hrm...  

That could be a problem, I'll have to rethink that, or just leave the maps out in a worst-case scenario.